AmazonChimeSDK/AmazonChimeSDK.xcodeproj/project.pbxproj

```diff
@@ -304,6 +305,20 @@
 		};
 /* End PBXContainerItemProxy section */
 
+/* Begin PBXCopyFilesBuildPhase section */
```

Comment:
Media SDK should be `Do not embed` instead of `Embed & Sign` for `AmazonChimeSDK` target.